feature-set-interface/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![cfg_attr(feature = "frozen-abi", feature(min_specialization))]
+#![deprecated(since = "4.0.1", note = "DO NOT USE. crates that need to break consesus should expose their own minimal configuration type")]
```

Comment:
typo in case we care
```suggestion
#![deprecated(since = "4.0.1", note = "DO NOT USE. Crates that need to break consensus should expose their own minimal configuration type")]
```